SPECIFICATION.md

```diff
@@ -63,61 +123,223 @@ The minimum difficulty that the pool supports. This will also be the default tha
-            "size": 32,
-            "proof": "0xb2cd6374c8db249ad3b638199dbb6eb9eaefe55042cef66c43cf1e31161f4a1280455d8b53c2823c747fd4f8823c44de3a52cc85332431630857c359935660c3403ae3a92728d003dd66ef5966317cd49894d265a3e4c43f0530a1192874ed327e6f35862a25dfb67c5d0d573d078b4b8ba9bfb1cce52fd17939ae9d7033d3aa09d6c449e392ba2472a1fecf992abcc51c3bf5d56a72fef9900e79b8dba88a5afc39e04993325a0cd6b67757355b836f"
-        },
-        "sp_hash": "0x4c52796ca4ff775fbcdac90140c12270d26a37724ad77988535d58b376332533",
-        "end_of_sub_slot": false,
-        "suggested_difficulty": 10,
         "launcher_id": "0xae4ef3b9bfe68949691281a015a9c16630fc8f66d48c19ca548fb80768791afa",
-        "owner_public_key": "0x84c3fcf9d5581c1ddc702cb0f3b4a06043303b334dd993ab42b2c320ebfa98e5ce558448615b3f69638ba92cf7f43da5",
-        "pool_payout_instructions": "c2b08e41d766da4116e388357ed957d04ad754623a915f3fd65188a8746cf3e8",
-        "authentication_key_info": {
-            "authentication_public_key": "0x970e181ae45435ae696508a78012dc80548c334cf29676ea6ade7049eb9d2b9579cc30cb44c3fd68d35a250cfbc69e29",
-            "authentication_public_key_timestamp": 1621854388
-        }
+        "authentication_token": 27062279,
+        "authentication_public_key": "0x970e181ae45435ae696508a78012dc80548c334cf29676ea6ade7049eb9d2b9579cc30cb44c3fd68d35a250cfbc69e29",
+        "payout_instructions": "0xc2b08e41d766da4116e388357ed957d04ad754623a915f3fd65188a8746cf3e8",
+        "suggested_difficulty": 10
     },
-    "auth_key_and_partial_aggregate_signature": "0xa078dc1462bbcdec7cd651c5c3d7584ac6c6a142e049c7790f3b0ee8768ed6326e3a639f949b2293469be561adfa1c57130f64334994f53c1bd12e59579e27127fbabadc5e8793a2ef194a5a22ac832e92dcb6ad9a0d33bd264726f6e8df6aad"
+    "signature": "0xa078dc1462bbcdec7cd651c5c3d7584ac6c6a142e049c7790f3b0ee8768ed6326e3a639f949b2293469be561adfa1c57130f64334994f53c1bd12e59579e27127fbabadc5e8793a2ef194a5a22ac832e92dcb6ad9a0d33bd264726f6e8df6aad"
 }
 ```
 
 Successful response:
 ```json
-{"points_balance": 1130, "current_difficulty": 10}
+{"welcome_message" : "Welcome to the reference pool. Happy farming."}
 ```
 
-The successful response must always contain the points balance since the last payout, as well as the current difficulty
-that this farmer is being given credit for.
+The successful response must always contain a welcome message which must be defined by the pool.
+
+#### payload
+
+#### payload.launcher_id
+The unique identifier of the farmer's singleton, see [Farmer identification](#farmer-identification).
+
+#### payload.authentication_token
+See [Farmer authentication](#farmer-authentication) for the specification of
+`authentication_token`.
+
+#### payload.authentication_public_key
+The public key of the authentication key, which is a temporary key used by the farmer to sign requests
+to the pool. It is an authorization given by the `owner_key`, so that the owner key can be potentially kept more secure.
```

Comment:
`can be potentially kept more secure` -> `can be kept more secure`

("more" is enough here)